src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -2010,12 +2009,6 @@ public static FunctionList getFunctions() {
     functions.add(
         new LibraryFunction("split_string", new AggregateType(DataTypes.STRING_TYPE, 0), params));
 
-    params = new Type[] {new AggregateType(DataTypes.STRING_TYPE, 0)};
```

Comment:
No. This is not OK.

You completely removed the two new "join_strings" ASH functions I recently added, as well as the unit tests.

Reply:
I'm sorry that I mistake to remove code because I was confused with the previous contents.